### DISCUSSION TITLE
license only allowing 30 downloads??? why?

### DISCUSSION THREAD

**Discussion by RollinCajun** [aclap-dev/video-downloadhelper#1294] [License & Purchase requests]
### What's your issue?

I have the purchased version. When I go to a page I want to download videos it will load and allow me to download the video each time I hit the play button but as soon as it downloads 30 videos it won't show the videos anymore when I click play. I have to reload the entire browser to reset the addon before it will let me again then I lose my place where I was downloading the videos and it is driving me nuts. Why is this limitation in place? It lets me do anywhere from 27-30 each time. It doesn't make any sense why this limit exists.

### The 10 first digits (only 10) of your license number or email address

ce1d69f4a7

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm not entirely certain this is a license issue.  In any case, instead of closing all your browser windows, you could go into the VDH settings & click the `Reload addon` button instead.  While you're in the VDH settings, click the `Copy information to clipboard` button, then paste that information into a post here.

I'm unclear what you mean by "not allow."  After you complete your approximately 30 successful downloads, what exactly happens?  Perhaps a couple of screenshots would clarify things.  What I'm wondering is whether the size of the VDH menu is being limited.  I'm not sure if that's what you are hitting or if VDH just won't recognize more than 30 videos on a page.  If the page in question is accessible without a user ID & password, you ought to post its URL.  If it is on a site that requires a registration, post a screenshot of the page.  Or maybe a few screenshots, scrolling the page, so we can get a sense of how the page is laid out.

**Reply by RollinCajun** [reply]
After clicking the download button on 27-30 downloads that show up on the addon as I hit the "play" button once it reaches that threshold it no longer shows the download button for any more of the videos when I hit play. I can hit play on 10 more videos and none of the downloads will show up in the addon. It's like it's capped at 30 per try. I've used other free downloaders that didn't do this so it's something to do with this extension. The site I'm getting it from is onlyfans and there is literally hundreds of videos well over 400. I had to use a free extension to get all the videos because this one was driving me stupid with the 30 cap. I never thought about reloading the extension I just kept reloading the entire browser and then having to scroll all the way down to where I left off by watching the last video and trying to find the thumbnail that matched. I'm not going to both even doing all what you said with the info etc because a free extension done exactly the same thing and didn't hassle me to death so I doubt I'll even use this one anymore. After the aggravation it caused me today I'm pretty done with it.
